ui/raidboss/popup-text.js

```diff
@@ -20,6 +20,16 @@ function onTriggerException(trigger, e) {
     console.error(lines[i]);
 }
 
+let params = new URLSearchParams(window.location.search);
```

Comment:
I think this logic seems reasonable, but I have a few small tweaks to suggest:

* is there a reason to have sounds and tts separate? It seems like maybe having a single `audio` param might be enough?
* maybe add a field on `Options` like `audioAllowed` that defaults to true and then if the url param is audio=0 or audio=1 then set it from there.  Then later on you can look up `this.options.audioAllowed` rather than have some free floating globals like `muteSounds`
* consolidate this URLSearchParams logic with the other code that uses it in the bottom of `ui/raidboss/raidboss.js`